.gitignore

```diff
@@ -0,0 +1,3 @@
+/node_modules
+/client/dist/bundle.js
```

Comment:
FYI you added bundle.js to your gitconfig, but you still committed it; by removing your bundle.js in the future, you can make your PR even smaller (by 296 lines of code, in this case)